### DISCUSSION TITLE
saving records into tables

### DISCUSSION THREAD

**Discussion by jchuillier** [dbeaver/dbeaver#21585] [DBeaver Development]
Hello all,
I'm just starting this so please excuse the mistakes.
I'm creating a SIMPLE db in postgre, using SIMPLE tables
And i can't save the effin records in the table !!!
I generate my sql insert into, all is well and when i go on the left pane, clic on my table, view and then to data the records are NOT there
I played with autocommit, manual commit, hit the save button on the left pane of the table, I see the table is 16ko so SOMETHING MUST be in there but i can never see it back
So what do i do wrong ?
Thanks for the help

**Comment by LonwoLonwo** [comment]
Hello @jchuillier 

Please provide a video of how you are adding and saving data.

**Comment by jchuillier** [comment]
Hello,
Thanks fr the tip, i got fed up and started everything with pgadmin and it worked flawlessly and immediatly, so something in the dbevaer initial config did not work for me.
